homeassistant/components/overkiz/coordinator.py

```diff
@@ -88,57 +91,8 @@ async def _async_update_data(self) -> dict[str, Device]:
         for event in events:
             _LOGGER.debug(event)
 
-            if event.name == EventName.DEVICE_AVAILABLE:
-                self.devices[event.device_url].available = True
-
-            elif event.name in [
-                EventName.DEVICE_UNAVAILABLE,
-                EventName.DEVICE_DISABLED,
-            ]:
-                self.devices[event.device_url].available = False
-
-            elif event.name in [
-                EventName.DEVICE_CREATED,
-                EventName.DEVICE_UPDATED,
```

Comment:
We don't seem to handle these events now.

Reply:
e1f13a79d65d6c97e776741f871c07d3d200184c